packages/vscode-extension/src/envTree.ts

```diff
@@ -76,11 +76,13 @@ export async function registerEnvTreeHandler(
       }
       await checkAllEnv(envNamesResult.value);
 
+      /*
```

Comment:
Add some comment to explain why this part is commented?

Reply:
Sure